content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

```diff
@@ -72,6 +72,11 @@ The upgrade workflow at high level is the following:
     {{< tabs name="k8s_install_kubeadm_first_cp" >}}
     {{% tab name="Ubuntu, Debian or HypriotOS" %}}
     # replace x in 1.17.x-00 with the latest patch version
+    apt-get update && \
+    apt-get install -y --allow-change-held-packages kubeadm=1.17.x-00
```

Comment:
we shouldn't have separate tabs for sub-flavors of a distro. if similar happens to RHEL we need to have yet another tab. also it is not clear what "older" means and the version of apt-get has to be explicitly pointed out.

here is how i think this single "Ubuntu or Debian" tab should look like:
```
old method
# since apt-get version x.xx you can also use the following:
new method
```